packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatBackedCallAdapter.ts

```diff
@@ -81,8 +83,18 @@ export class CallWithChatBackedCallAdapter implements CallAdapter {
     return this.callWithChatAdapter.joinCall(microphoneOn);
   };
   public leaveCall = async (): Promise<void> => await this.callWithChatAdapter.leaveCall();
-  public startCall = (participants: string[], options: StartCallOptions): Call | undefined => {
-    return this.callWithChatAdapter.startCall(participants, options);
+  public startCall = (
+    participants: string[] | CommunicationIdentifier[],
+    options: StartCallOptions
+  ): Call | undefined => {
+    const communicationParticipants: CommunicationIdentifier[] = participants.map((participant) => {
```

Comment:
nit: use `_toCommunicationIdentifier` here
```suggestion
    const communicationParticipants = participants.map((_toCommunicationIdentifier));
```

Reply:
and `removeParticipant` below